ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in another reward-bearing AF program
+- Commit the rewards to an xGov DAO for a period
+ 
+The foundation will maintain lists of eligible reward-bearing programs, and a list of qualified DAOs.
+ 
+#### Example
+Alice participates in governance during Q1 2023
+the potential reward is 5 Algo, Alice:
+1) Commit 100 Algo in an AF program
+2) Express her will to participate in xGov
+3) Will not receive 5 Algo at the end of Q1 2023. They are instead locked in one big xGov pool for 1 year
+4) Will receive 5 Algo at Q1 2024
+ 
+## Rationale
+In this ARC we seek community input on issues such as, but not limited to:
+- The types of reward-bearing activities that **SHOULD** be included in the qualification list (ie. general governance)
+- Whether xGovs **MUST** vote on all measures to keep their status
+- Whether xGovs tokens **SHOULD** be sold/transferred or **SHOULD** be burned when exiting the xGov DAO
+- Whether xGovs tokens expire after a year or gain more voting power if the stake remains in the DAO
+- Whether there **SHOULD** be a minimum and a maximum amount of Algos required to become an xGov
+- Whether there be a maximum amount of Algos that can be locked by xGovs for the each period, limiting the total number of xGovs
+- Whether xGovs **SHOULD** be disqualified and lose their status if they behave in a suboptimal fashion
+- What types of suboptimal behavior would trigger disqualification or loss of status?
```

Comment:
Slashing is only acceptable if it's fully decentralized with a large majority in my opinion. If we do this I would not create rules in advance but allow xGov proposals to disqualify other xGov members with a big majority of votes.

Even if we allow this what is stopping the xGov member to join again (maybe with another wallet)? Do we plan to have any kind of doxxing?

Reply:
yea I'm just not sure how you're going to police this by creating issues of centralization in itself if you give the AF foundation the ability to pick and choose suspensions in some sort of process that isn't completely written in code.

Reply:
Less than 90% participation node uptime during any quarter or less than 95% uptime during a 1 year period. 

Failing to vote at any time in quarter. 

Failing to comment on X number of proposals in a 1 year period (we want proactive participants).

Reply:
Or instead of uptime you have to vote on consensus X amounts per month. I don't know the odds, but even with 1 ALGO it's likely to happen with some percent chance. This would be much easier to measure I think.